pkg/addons/data/ConfigMap/openshift-azure-logging/mdsd-config.yaml

```diff
@@ -7,14 +7,11 @@ data:
             <Account moniker="_ACCOUNT_" isDefault="true" autoKey="false" />
         </Accounts>
         <Management eventVolume="Large" defaultRetentionInDays="90" >
-            <Identity tenantNameAlias="RPTenant" roleNameAlias="Service" roleInstanceNameAlias="Host">
-                <IdentityComponent name="RPSector">_SECTOR_</IdentityComponent>
-                <IdentityComponent name="RPTenant">_REGION_</IdentityComponent>
-                <IdentityComponent name="Service">OSA</IdentityComponent>
-                <IdentityComponent name="Host" useComputerName="true" />
+            <Identity tenantNameAlias="ResourceName" roleNameAlias="ResourceGroupName" roleInstanceNameAlias="SubscriptionId">
```

Comment:
it is not clear to me that this is valid - do these need to be OSA service identifiers?

Reply:
For this, we need to discuss. I would like this to confirm with MSFT.
This is 3 default identities for Jarvis. We can disable use of defaults and don't have these.